### PR TITLE
Storytelling logos

### PR DESCRIPTION
**Pull request by polyrand**
We've created a few logos representing "emotions". We think they can be fun to have inside the diagrams.

### CODE REVIEW

**Comment by lipis** [comment]
Since it's from `drwnio` could you please add them there instead of creating a new director? That way all libs from the same author are in the same place.

File: libraries.json [hunk 1]
```diff
@@ -178,5 +178,14 @@
     "source": "maeddes/technology-logos.excalidrawlib",
     "preview": "maeddes/technology-logos.png",
     "date": "2021-01-27"
+  },
+  {
+    "name": "Storytelling",
+    "description": "Some drawings representing emotions or whatever you want. Perfect to add some fun or stories to your diagrams.",
```
Comment (lipis) [inline libraries.json:184]:
```suggestion
    "description": "Drawings representing emotions or whatever you want. Perfect to add some fun or stories to your diagrams.",
```

**Comment by polyrand** [comment]
Sure! I've just moved them :D

**Comment by lipis** [comment]
Awesome.. thanks!

**Comment by lipis** [comment]
Live in 2 minutes

**Pull request merged**